domains/ridwan.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ridwanyinus",
+           "email": "ridwanyinusa15@gmail.com",
+           "discord": "1142082910351011991"
+        },
+    
+        "record": {
+            "CNAME": "https://ridwanyinus.vercel.app/"
```

CNAME is invalid and we currently don't support Vercel
```suggestion
            "CNAME": "ridwanyinus.vercel.app"
```